test/fakefs_file_test.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'test_helper'
```

Comment:
is there no other test that already covers file.rb ?

Reply:
I saw an explicit test file for `FaksFS::FakeFile` but not `FakeFS::File`. Any idea where it might be?

Reply:
Ah, yea it seems that some specs are present in the `FakeFSTest`. Would you prefer I move this test into that file (`/test/fakefs_test.rb`)?

Reply:
yeah, prefer to have 1 file then piecemeal ... ideally extract all the file tests to a single test-file

Reply:
> ideally extract all the file tests to a single test-file

Will do!